### ISSUE TITLE
Annotation: Text Position with Wrong Spacing

### ISSUE THREAD

**Issue by LerkLin** [ScottPlot/ScottPlot#3749]
**Issue:** Text Render in Annotation usiing Incorrect Spacing

**ScottPlot Version:** 5.0.31

**Code Sample:** 
Extra space area like below:
```cs
        var plt = new ScottPlot.Plot();
        plt.Add.Signal(ScottPlot.Generate.Sin());
        var strDisplay = new StringBuilder();
        foreach(var i in Enumerable.Range(1, 10))
        {
            strDisplay.AppendLine($"this is {i} Line");
        }

        plt.Add.Annotation(strDisplay.ToString());
        plt.SavePng("plt.png",300,500);
```
![plt](https://github.com/ScottPlot/ScottPlot/assets/52663844/4013b4b5-6680-4150-a502-7234f75798ce)

**Comment by swharden** [comment]
Thanks for reporting this @LerkLin! I'm going to try to make a quick fix now, but a true fix will come when #3700 is properly implemented

**Comment by swharden** [comment]
Oof, fixing the label measurement for annotations breaks it for axis tick labels 🤦 

I have to put this down for now, but I'll pick it up later today! Thanks again for reporting this 👍

**Comment by swharden** [comment]
Fixed in #3750 🚀

**Comment by LerkLin** [comment]
@swharden appreciate that 😃
